platform/openide.util/test/unit/src/org/openide/util/TaskTest.java

```diff
@@ -260,26 +304,32 @@ public void testWaitWithTimeOutReturnsAfterTimeOutWhenTheTaskIsNotComputedAtAll
         
         fail ("Something wrong happened the task should wait for 1000ms but it took: " + time + "\n" + log);
     }
+    
+    //--------------------------------------------------------------------------
     @Test
-    public void testWaitOnStrangeTaskThatStartsItsExecutionInOverridenWaitFinishedMethodLikeFolderInstancesDo () throws Exception {
+    public void waitOnTask_thatStartsItsExecutionWithOverridenWaitFinishedMethod() 
```

Comment:
the test case methods are getting _really_ long. I personally would prefer having waitOnTask1-x  while putting the rest into the javadoc for the method.

I believe junit 5 (which we don't use) has a description annotation. This would also make it easier one day to move the doc comment into an annotation if someone decides to upgrade the testing lib.

Reply:
I try to follow the convention "what_behavesHow_underWhatConditions" which I adopted from [Kevlin Henney](https://youtu.be/3frRGQGiPuc?t=2829) - it sorts incely in IDEs class view.
yes - the test methods bacome longer but also more descriptive (though putting some of this description ind annotation might be good).
the reliefing factor is that nobody call these methods directly :)

Reply:
yes but this is simply too long - it doesn't scale. They also do appear in test output windows etc and this makes it hard to read. I would just give up reading after the second compound word and look at the code instead. Esp since some of the names are longer then the test code - its just noise.

`waitOnTask` with a short description if necessary makes it more useful and is also closer to the style already in other tests. Please try to find some middle ground and don't use method name as a full sentence.

Reply:
I shoretened some method names in https://github.com/apache/netbeans/pull/5061/commits/a1d38f89757c486e5c8f9a2e57d28d0a75585068
I'll try to keep a soft 50 character length limit for test method names in the future.

Reply:
ok thanks. looks better. Feel free to squash and force push. Lets get this finally in.

Reply:
@matthiasblaesing @neilcsmith-net are you two also ok with this long-form method naming style for tests? I won't want to be the boo man here and block this for style reasons.

Reply:
squashed